command/operator_migrate.go

```diff
@@ -44,6 +45,7 @@ type migratorConfig struct {
 	StorageSource      *server.Storage `hcl:"-"`
 	StorageDestination *server.Storage `hcl:"-"`
 	ClusterAddr        string          `hcl:"cluster_addr"`
+	MaxParallel        string          `hcl:"max_parallel"`
```

Comment:
`max_parallel` might cause a bug with the `start` flag. Could we please make `max_parallel` a flag instead of a config option and error if both these flags are specified?

Reply:
Removed in favor of a command flag.